dotcom-rendering/src/components/Footer.tsx

```diff
@@ -332,9 +332,8 @@ export const Footer = ({
 	>
 		<div css={pillarWrap}>
 			<Pillars
-				display={ArticleDisplay.Standard}
 				pillars={pillars}
-				pillar={pillar}
+				selectedPillar={themeToPillar(theme)}
```

Comment:
If I understand this correctly, there are a couple of scenarios where this would decide a on selected pillar that's different from the one a given piece has set?

1. For an article with a special theme (labs, special report, special report alt), it will ignore any existing pillar and highlight `news`
2. For an article with no pillar, it will highlight `news`

In scenario 1) the behaviour is different from the header nav, which uses `getCurrentPillar` rather than `themeToPillar` to derive the `selectedPillar`?

Regarding 1), would we want to make sure that the header nav and footer nav have the same thing selected for a given article? If so, I think we have the choice of deriving this from the article's original pillar, as the header nav does, or using `themeToPillar` as the footer nav does. I suspect the header nav behaviour would be the correct one, but what do you think?

Regarding 2), are we sure we want to mark pieces with `news` when they haven't been specified as being part of that pillar? Would it be more accurate to say that they don't live under any pillar, and not highlight anything?

I'm not sure I've described what I mean above very well 😬, so here are a couple of examples to try and clarify.

## Scenario 1 - Existing Pillar

This piece has a special report alt theme, and is published under the opinion pillar: https://www.theguardian.com/commentisfree/2023/mar/28/the-guardian-view-on-our-links-to-slavery-a-necessary-reckoning-with-the-past

In PROD this has "opinion" selected in the header nav, and nothing selected in the footer nav. With this change this has "opinion" selected in the header nav, and "news" selected in the footer nav.

| | Before (PROD) | After (Change) |
| - | - | - |
| Header | ![header-prod] | ![header-change] |
| Footer | ![footer-prod] | ![footer-change] |

[header-change]: https://github.com/guardian/dotcom-rendering/assets/53781962/51360975-14ac-4f20-a1dd-8649a392cff9
[header-prod]: https://github.com/guardian/dotcom-rendering/assets/53781962/ebcebaf7-7bcb-49b7-9185-29c3e5305ed7
[footer-prod]: https://github.com/guardian/dotcom-rendering/assets/53781962/f02d3012-f2a2-480a-b32f-d6f5dedc6a76
[footer-change]: https://github.com/guardian/dotcom-rendering/assets/53781962/74a0869d-361d-4536-acbd-7f0966ed440d

## Scenario 2 - No Pillar

This piece has a labs theme, and is published with no pillar: https://www.theguardian.com/a-syllabus-for-saving-the-world/2023/jun/16/behavioural-battlefronts-why-the-transition-to-a-zero-carbon-economy-needs-psychology-as-well-as-technology

In PROD this has nothing selected in the header or footer navs. With this change this has "news" selected in the header and footer navs.

| | Before (PROD) | After (Change) |
| - | - | - |
| Header | ![header-prod-labs] | ![header-change-labs] |
| Footer | ![footer-prod-labs] | ![footer-change-labs] |

[footer-prod-labs]: https://github.com/guardian/dotcom-rendering/assets/53781962/7fdcb77c-faf2-429c-8782-501663251eca
[header-change-labs]: https://github.com/guardian/dotcom-rendering/assets/53781962/677609c2-5f0c-42bc-afb3-2b0efb40e258
[header-prod-labs]: https://github.com/guardian/dotcom-rendering/assets/53781962/f34c1b4d-3f29-464f-9ca9-27a15513d022
[footer-change-labs]: https://github.com/guardian/dotcom-rendering/assets/53781962/e0c995e1-9b80-4639-bc63-a39f02b59124

Reply:
Thank you for this, very clear. 

1) definitely we want header and footer to agree
2) you're right we should probably allow for undefined 

Let me have a think what the best way to do this is